src/app/footer/footer.component.html

```diff
@@ -56,16 +56,17 @@
       <div class="link-column">
         <div class="header">Stay in touch</div>
         <div class="social-links">
-          <a href="https://www.instagram.com/codeforafrica">
+          <a href="https://twitter.com/Code4Africa" target="_blank">
```

Comment:
If `target="_blank"`, you need `noreferrer noopener` as well, no?

Reply:
This is not resolved. Where is `rel` @kelvinkipruto ?

Reply:
For all links @kelvinkipruto. Background: https://web.dev/external-anchors-use-rel-noopener/